### DISCUSSION TITLE
Can we start using the ui library in production?

### DISCUSSION THREAD

**Discussion by skwowet** [heroui-inc/heroui#977] [General]
We're building an open source SaaS product, and currently looking for a stable UI library to stick with for a long run. Love the next-ui library, but still concerned about choosing it.

(we're okay with few bugs here and there which majorly doesn't affect our work)

**Comment by Eppu** [comment]
NextUI is still in beta, so I'd be careful about using it in production. I haven't run into issues with my personal projects, but your mileage may vary.

As stated in `README.md`:
>Warning 🚧: This project is still in development, it might have bugs and breaking changes so it is not recommended to use it in production.
